kinetic/distribution.yaml

```diff
@@ -13251,6 +13251,21 @@ repositories:
       url: https://github.com/tork-a/roswww.git
       version: develop
     status: maintained
+  rowma_ros:
+    doc:
+      type: git
+      url: https://github.com/rowma/rowma_ros.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/rowma/rowma_ros.git
```

Comment:
I just noticed this, but we highly discourage using the source repository as the release repository.  The reason is that bloom tends to drop a lot of branches and other stuff into the release repository that just isn't needed when you are doing from-source development.  I'll suggest you create a separate repository called `rowma_ros-release`, and then re-release into there.